priv/scripts/elixir.sh

```diff
@@ -80,18 +80,19 @@ function upload_build {
 function update_builds_txt {
   otp_versions=(${@:3})
   date=$(date -u '+%Y-%m-%dT%H:%M:%SZ')
+  build_sha256=$(sha256sum mix.exs | cut -d ' ' -f 1)
```

Comment:
we need to checksum the zip which is saved as `elixir.zip`:

```suggestion
  build_sha256=$(sha256sum elixir.zip | cut -d ' ' -f 1)
```

but there's a problem, more on this below.